### PR TITLE
Add information in documentation about patching by environment variable

### PR DESCRIPTION
**Pull request by Pahandrovich**

### CODE REVIEW

File: doc/sklearn.rst [hunk 1]
```diff
@@ -22,6 +22,18 @@ scikit-learn application by using the following commandline flag::
 
     python -m daal4py my_application.py
 
+To enable daal4py patches through an environment variable, you can set the environment variable.
```
Comment (outoftardis) [inline doc/sklearn.rst:25]:
```suggestion
To enable daal4py patches through an environment variable, set ``USE_DAAL4PY_SKLEARN`` to 1 as shown below.
```

File: doc/sklearn.rst [hunk 2]
```diff
@@ -22,6 +22,18 @@ scikit-learn application by using the following commandline flag::
...
+
+On Linux and Mac OS::
+
+    export USE_DAAL4PY_SKLEARN=1
+
+On Windows::
+
+    set USE_DAAL4PY_SKLEARN=1
+
+To disable, set the environment variable to 0.
```
Comment (outoftardis) [inline doc/sklearn.rst:35]:
```suggestion
To disable daal4py patches, set the ``USE_DAAL4PY_SKLEARN`` environment variable to 0.
```

**Comment by Alexander-Makaryev** [comment]
Variable values may vary:
if _os.environ.get('USE_DAAL4PY_SKLEARN', False) in [True, '1', 'y', 'yes', 'Y', 'YES', 'Yes', 'true', 'True', 'TRUE'].

This feature will works only with our version of Scikit-Learn.
if you install Scikit-Learn from conda-forge or somewhere else, it will not take effect.

File: doc/sklearn.rst [hunk 3]
```diff
@@ -22,6 +22,20 @@ scikit-learn application by using the following commandline flag::
 
     python -m daal4py my_application.py
 
+If using scikit-learn from intel conda channel, then
```
Comment (outoftardis) [inline doc/sklearn.rst:25]:
```suggestion
If you are using scikit-learn from Intel conda channel, then 
```

wouldn't `Intel(R) Distribution for Python` be a correct way to name "scikit-learn from Intel conda channel"?

File: doc/sklearn.rst [hunk 4]
```diff
@@ -22,6 +22,20 @@ scikit-learn application by using the following commandline flag::
 
     python -m daal4py my_application.py
 
+If using scikit-learn from intel conda channel, then 
+you can enable daal4py patches through an environment variable, set ``USE_DAAL4PY_SKLEARN`` to one of the values
+``True``, ``'1'``, ``'y'``, ``'yes'``, ``'Y'``, ``'YES'``, ``'Yes'``, ``'true'``, ``'True'``, ``'TRUE'`` as shown below.
```
Comment (outoftardis) [inline doc/sklearn.rst:27]:
```suggestion
you can enable daal4py patches through an environment variable. To do this, set ``USE_DAAL4PY_SKLEARN`` to one of the values
``True``, ``'1'``, ``'y'``, ``'yes'``, ``'Y'``, ``'YES'``, ``'Yes'``, ``'true'``, ``'True'``, or ``'TRUE'`` as shown below.
```

**Comment by Pahandrovich** [comment]
@Alexander-Makaryev, @outoftardis, can you look this PR again?

File: doc/sklearn.rst (hunk 3 above)
Comment (Pahandrovich) [reply doc/sklearn.rst:25]: This is a good question, I will ask about it

Comment (Pahandrovich) [reply doc/sklearn.rst]: Fixed

File: doc/sklearn.rst (hunk 4 above)
Comment (Pahandrovich) [reply doc/sklearn.rst]: Fixed

**Comment by Pahandrovich** [comment]
@outoftardis, @Alexander-Makaryev, any other concerns?

**Comment by outoftardis** [comment]
I don't have any, looks good to me

**Pull request merged**